dfastmi/gui.py

```diff
@@ -332,13 +333,15 @@ def updated_reach(ireach: int) -> None:
     """
     ibranch = dialog["branch"].currentIndex()
 
-    q_stagnant = rivers.qstagnant[ibranch][ireach]
+    reach = rivers.branches[ibranch].reaches[ireach]
+    q_stagnant = reach.qstagnant
     dialog["qthr"].setText(str(q_stagnant))
-    dialog["ucrit"].setText(str(rivers.ucritical[ibranch][ireach]))
-    update_qvalues()
+    ucritical = reach.ucritical
+    dialog["ucrit"].setText(str(ucritical))
+    update_qvalues(reach)
 
 
-def update_qvalues() -> None:
+def update_qvalues(reach:ReachAdvanced) -> None:
```

Comment:
nice! here you pass a reach!

Reply:
I agree, but gui.py is not in scope of this issue. I only refactor code to get it working again. Will pickup the good comments later in new issue

Reply:
so basicly you just copied and pasted the comment in each comment I made?

Reply:
euh, well i did read everything but came to the same answer about the expected scope. Maybe mikado, only work on what is needed for the issue, get the rest working again and pick/create new issues if needed